main/src/library/SetMultiMap.flix

```diff
@@ -0,0 +1,514 @@
+/*
+ * Copyright 2022 Stephen Tetley
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+///
+/// The SetMultiMap type.
```

Comment:
I think we should rename this to `MultiMap`. That's the standard name, imo.

Reply:
Fixed. Now MultiMap (and Bag MultiMap removed from PR).

Reply:
Thanks!